.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ commands:
             # Currently downloading form our own buckets due to:
             # https://github.com/emscripten-core/emscripten/issues/14987
             #wget -O ~/chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-            wget -O ~/chrome.deb https://storage.googleapis.com/webassembly/chrome/google-chrome-stable_current_amd64_new.deb
+            wget -O ~/chrome.deb https://storage.googleapis.com/webassembly/chrome/google-chrome-stable_current_amd64.deb
```

Comment:
I think we should just delete this line and uncomment the line above no?  So that we continue to track chrom stable?

Reply:
And we should speculatively close https://github.com/emscripten-core/emscripten/issues/14987

Reply:
Maybe, but I think keeping it pinned is probably a good idea to prevent unexpected changes from breaking us. Maybe we can set up an autoroller to automatically update the version of chrome in the storage bucket periodically?

Reply:
Our policy historically has been to expose our selves to the latest stable chrome and firefox and then deal with issues that come up.  We can discuss possibly changing that .. but setting up and auto-roller sounds complicated.  @kripken ?

Reply:
I think given the comment above and the link the bug it makes sense to remove the comment and restore the old status quo.

Reply:
I really think pinning is not the right thing here.

If any change to Chrome breaks us then that is a major event we do not want to miss. It may be a bug in Chrome that we need to file asap - it may affect more than Emscripten, so this is a larger issue than us - or it may be an urgent Emscripten change we need to make in response to an upstream change (like say when SAB was moved behind COOP/COEP).

The inconvenience to us of things breaking on CI now and then is negligible compared to the benefit of finding a problem with latest Chrome and preventing that problem from affecting our users.

In fact we really should be using beta here, to give us a chance to actually have a chance of bugs getting fixed before final releases. We used to be on beta until https://github.com/emscripten-core/emscripten/pull/14370 - I think we should go back to beta immediately.